bloomrun.js

```diff
@@ -43,6 +43,40 @@ BloomRun.prototype.add = function (pattern, payload) {
   return this
 }
 
+BloomRun.prototype.remove = function (pattern) {
+  var properties = this._properties
+  var matches = matchingBuckets(this._buckets, pattern)
+  var buckets = this._buckets
+  var bucket
+
+  if (matches.length > 0) {
+    bucket = matches[0]
+
+    for (var i = 0; i < bucket.data.length; i++) {
+      if (pattern === bucket.data[i].pattern) {
+        bucket.data.splice(i, 1)
+      }
```

Comment:
Given the current API, you can add two elements with the same pattern, but different payload. `remove(payload)` should really be `remove(pattern, payload)`.